app/decorators/website_decorator.rb

```diff
@@ -24,4 +24,45 @@ def contact_email
   def closes_at
     event.closes_at(:month_day_year)
   end
+
+  def session_format_configs
+    event.session_formats.map.with_index do |session_format, index|
+      SessionFormatConfig.find_or_initialize_by(session_format: session_format) do |config|
+        config.name = session_format.name
+        config.position = index + 1
+      end
+    end
```

Comment:
It may be nice to add a `.sort_by(&:position) to this so that they get reordered by their position.